client/signup/config/flows-pure.js

```diff
@@ -453,12 +453,12 @@ export function generateFlows( {
 		{
 			name: 'woocommerce-install',
 			pageTitle: translate( 'Add WooCommerce to your site' ),
-			steps: [ 'select-site', 'confirm', 'transfer' ],
+			steps: [ 'confirm', 'transfer', 'select-site' ],
 			destination: '/',
 			description: 'Onboarding and installation flow for woocommerce on all plans.',
-			providesDependenciesInQuery: [ 'siteSlug' ],
-			optionalDependenciesInQuery: [ 'siteSlug' ],
-			disallowResume: true,
+			providesDependenciesInQuery: [ 'site' ],
```

Comment:
It's a shame we can't use `siteSlug` as it seems to be handled in much the same way. Does it work or does the controller stuff break things?

Reply:
signupDependencies.siteSlug takes precendence over queryObject.siteSlug and talking to @p-jackson we weren't comfortable changing this around. Adding queryObject.site that takes precedence over signupDependencies.site and siteSlug/siteId seemed like the safest option.